image.yaml

```diff
@@ -3,8 +3,10 @@ schema_version: 1
 name: "redhat-sso-7/sso72"
 description: "Red Hat Single Sign-On 7.2 container image"
 version: "7.2.0"
-from: "jboss-eap-7/eap71:7.1.0-9"
+from: "jboss-eap-7/eap71:latest"
```

Comment:
@goldmann, do we want to stick with latest, or should we pin this to the latest released version, e.g. 7.1.1, to ensure we get the latest released version and not just the latest version.

Reply:
You can pin to latest released image (check with EAP team which one is it) to be sure which image you use. By using `latest` you have no control over which image you pull now. This will change in the future slight;y.